tests/test_client.rs

```diff
@@ -506,3 +506,31 @@ fn client_read_until_eof() {
     let bytes = sys.block_on(response.body()).unwrap();
     assert_eq!(bytes, Bytes::from_static(b"welcome!"));
 }
+
+#[test]
+fn client_basic_auth() {
+    let mut srv =
+        test::TestServer::new(|app| app.handler(|_| HttpResponse::Ok().body(STR)));
+
+    let request = srv
+                    .get()
+                    .basic_auth("username",Some("password"))
```

Comment:
Space after `,`

Reply:
@DoumanAsh 

- Updated the spacing.
- Added comments above the requests in test_client.rs.
- rebase.